packages/fiori/src/ShellBar.hbs

```diff
@@ -54,7 +55,7 @@
 		{{/if}}
 
 		{{#if secondaryTitle}}
-			<h2 class="ui5-shellbar-secondary-title">{{secondaryTitle}}</h2>
+			<h2 class="ui5-shellbar-secondary-title" data-ui5-stable="secondaryTitle">{{secondaryTitle}}</h2>
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** I'm seeing in the following...

https://github.com/SAP/ui5-webcomponents/blob/b81e4e16511a85508593fbccd2241eca5cdf1c9a/packages/fiori/src/ShellBar.hbs#L174

...that kebab case is being used for the value of ` data-ui5-stable`.  So should we do the same here?:

```suggestion
			<h2 class="ui5-shellbar-secondary-title" data-ui5-stable="secondary-title">{{secondaryTitle}}</h2>
```